CHANNGELOG.md

```diff
@@ -2,7 +2,8 @@ authconfig Cookbook Changelog
 =============================
 This file is used to list changes made in each version of the authconfig cookbook.
 
-v1.1.13
--------
-### BUG
-- Avoid sssd start or restart if sssd daemon disabled
+## 2.0.5
+* Enable/start sssd if it is being installed
```

Comment:
A config change there should initiate a delayed restart, which shouldn't affect the enable/start that I added.

Primary issue we have is that sssd isn't always being started on VMs and the way this recipe is configured the only attempts made at restarts happen during config changes, so if it doesn't start the first time, Chef won't try to start it again unless the config changes.